array-checker.js

```diff
@@ -0,0 +1,13 @@
+function uniqueCalc(arr1, arr2) {
+  let unique1 = arr1.filter((o) => arr2.indexOf(o) === -1);
+  let unique2 = arr2.filter((o) => arr1.indexOf(o) === -1);
+  let sum = 0;
+  const unique = unique1.concat(unique2);
+
+  for (let i = 0; i < unique.length; i++) {
```

Comment:
In such cases where you don't need the index and are looping over the entire array, `for..of` could be a nice abstraction:

```js
for (const num of unique) {
  sum += num;
}
```